manage.py

```diff
@@ -15,11 +16,20 @@ class Development(LocalRoot):
 @Development.register
 @local('remainder', metavar='alembic arguments', nargs=argparse.REMAINDER)
 def alembic(context, args):
-    """Configuration wrapper to use the Alembic schema migrations library for Triage development"""
+    """Configuration wrapper to use the Alembic schema migrations library for Triage development.
+    Try alembic -h to see description of all available subcommands"""
     return context.local['env'][
         'PYTHONPATH=' + str(ROOT_PATH / 'src'),
         'alembic',
         '-c', ROOT_PATH / 'src' / 'triage' / 'component' / 'results_schema' / 'alembic.ini',
         '-x', 'db_config_file=database.yaml',
         args.remainder,
     ]
+
+
+@Development.register
+class Docs(Local):
+    """View Triage documentation through local server"""
+    def prepare(self, args):
+        with plumlocal.cwd(plumlocal.cwd / 'docs'):
```

Comment:
Yeah the `local` wrapper that `argcmdr` provides should probably expose more stuff, like the handy `cwd` context manager; but, it doesn't, currently :man_shrugging: